packages/viewer/stencil.config.ts

```diff
@@ -11,11 +10,10 @@ import jestConfig from './jest-shared.config';
 
 export const config: Config = {
   namespace: 'viewer',
-  nodeResolve: {
-    browser: true,
-  },
+  sourceMap: true,
+  nodeResolve: { browser: true },
+  preamble: copyright(),
```

Comment:
Nice to remove the dependency on this custom plugin - was this messing with the sourcemaps?